config/settings/base.py

```diff
@@ -124,7 +124,7 @@
 
 USE_TZ = True
 
-GA_TAG_KEY = env.str("GOOGLE_GA_TAG_KEY", "REPLACE_ME")
+GA_TAG_KEYS = env.str("GOOGLE_GA_TAG_KEYS", "REPLACE_ME")
```

Comment:
There's an `env.list` which will handle splitting a `,` separated value into a list for us.

Reply:
We can probably also change the default to an empty list `[]`, then we can get rid of the `if` in the template